aw/Extensions/UIView.swift

```diff
@@ -0,0 +1,38 @@
+import UIKit
+
+extension UIView {
```

Comment:
What's the purpose of this extension?

Reply:
Trying to make things a little cleaner to do programmatically, but then I ended up having to make big changes in IB anyways (View Controller with TableView rather than TableViewController), so I ended up building it that way in the end.

Reply:
It's removed now.